apps/veil/src/pages/explore/ui/waves.tsx

```diff
@@ -7,7 +7,8 @@ export const PenumbraWaves = () => {
   return (
     <Waves
       className={cn(
-        'display-none absolute scale-90 w-[1200px] top-[-800px] left-[25%]  z-[-30] opacity-90',
```

Comment:
@vacekj question: why did you change it before? In my visual tests, the update contradicts Figma designs and doesn't adapt to mobile and even small-desktop screens.